tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -526,6 +526,18 @@ def eval_if_tensor(x):
 
     if not isinstance(spec, (list, tuple)):
       spec = [spec]
+   
+    if (len(spec) > 1) and \
+      (all(isinstance(i, list) for i in spec) or \
+      all(isinstance(i, ops.Tensor) for i in spec) or \
+      all(isinstance(i, np.ndarray) for i in spec)):
+      if not (any(i is Ellipsis) for i in slice_spec):
+        np_specs = []
+        for s in spec:
+          np_specs.append(eval_if_tensor(s))
+        tensor = op.eval()
```

Comment:
Why is this block of code in the tests needed?

Reply:
Just want to successfully deal with `testFetchMulti` below.

Reply:
Just want to deal with the test `testFetchMulti` below.